.idea/gradle.xml

```diff
@@ -15,7 +15,6 @@
           </set>
```

Comment:
Please avoid pushing file changes for files from the idea folder next time.

Reply:
ok. Probably didn't see that